### ISSUE TITLE
New Rendering Backend

### ISSUE THREAD

**Issue by Antidote** [RetroView/MetPrimeTools#24]
This is a bit of a show stopper, Qt is great for handling the context, but it hides way to much.
We need to have a list of requirements for the new backend.

**Comment by jackoalan** [comment]
indeed.. first things first, figure out how to acquire a drawing surface where we need it (Qt may still need to hold the framebuffer, but rendering commands need not be dispatched by it)

**Comment by jackoalan** [comment]
http://doc.qt.io/qt-5/qtplatformheaders-module.html#classes
we establish our own contexts in platform-native terms and supply them to Qt this way

**Comment by jackoalan** [comment]
http://stackoverflow.com/a/18000819
that's how you make a QWidget from a QWindow (custom OGL context backed)

**Comment by Antidote** [comment]
Hmm, great food for thought, I'll definitely be reading up on those.
